PR adressing Roman's AD review of -04

emanjon opened the pull request.

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

emanjon commented:
#2

obgm commented on draft-ietf-ace-extend-dtls-authorize.md:
```diff
@@ -62,14 +62,14 @@ entity:
 # Introduction
 
-{{RFC9202}} only specifies the use of DTLS {{RFC6347}} {{RFC9147}} but works equally well for TLS {{RFC8446}}. For many constrained implementations, CoAP over UDP {{RFC7252}} is the first choice, but when deploying ACE in networks controlled by other entities (such as the Internet), UDP might be blocked on the path between the client and the RS, and the client might have to fall back to CoAP over ...
+{{RFC9202}} only specifies the use of DTLS {{RFC6347}} {{RFC9147}} but works equally well for TLS {{RFC8446}}. For many constrained implementations, CoAP over UDP {{RFC7252}} is the first choice, but when deploying ACE in networks controlled by other entities (such as the Internet), UDP might be blocked on the path between the client and the RS, and the client might have to fall back to CoAP over ...
```
The problem with this change is that "the dual support for TLS and DTLS" ist not supported by the OSCORE profile because OSCORE achieves its goals by being transport protocol agnostic. I was stuck on this for a while to come up with a more accurate description.

gselander replied:
How about "This dual support for security over TCP as well as UDP is already supported by the OSCORE profile {{RFC9203}}."

obgm replied:
This is fine by me.

gselander committed:
In terms of (TCP, UDP) instead of (TLS, DTLS)

gselander commented:
I think we are ready to merge then.

emanjon committed:
RFC9325

emanjon commented:
No the are still most of the review left to fix. Everything should should be fixed in the same PR. No reason to merge until the PR is complete.

John

From: Göran Selander ***@***.***>
Date: Tuesday, 13 December 2022 at 13:26
To: ace-wg/ace-extend-dtls-authorize ***@***.***>
Cc: John Mattsson ***@***.***>, Author ***@***.***>
Subject: Re: [ace-wg/ace-extend-dtls-authorize] Update draft-ietf-ace-extend-dtls-authorize.md (PR #3)

I think we are ready to merge then.

—
Reply to this email directly, view it on GitHub<https://protect2.fireeye.com/v1/url?k=31323334-501cfaf3-313273af-454445554331-2869f280301455e0&q=1&e=eeb00d9c-6b3d-4086-8c00-d2e9baa587ef&u=https%3A%2F%2Fgithub.com%2Face-wg%2Face-extend-dtls-authorize%2Fpull%2F3%23issuecomment-1348439216>, or unsubscribe<https://protect2.fireeye.com/v1/url?k=31323334-501cfaf3-313273af-454445554331-fee73780cfd015fe&q=1&e=eeb00d9c-6b3d-4086-8c00-d2e9baa587ef&u=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFYXKYUB3DB3F3N67CURPETWNBTQHANCNFSM6AAAAAASW56P6U>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

emanjon committed:
Keep RFC 9202 in IANA registry

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

emanjon committed:
Not part of AD review

RFC editor can fix abreviations later.

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

Reformulated text in section 1.
added Client and RS

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

Added 

It is up to the
implementation to handle the case where the RS reponds to both connection
requests.

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

Removed normative text related to error and state.

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

Clarified that establishment of the security association is to RS again

emanjon committed:
Update draft-ietf-ace-extend-dtls-authorize.md

Clarified that initial unauthorized resource request is to the RS.

obgm commented:
Thanks @emanjon and @gselander These changes look good to me (i.e., approval). No objections from my side for squash/merge.
